package_format/example_main_page/parameters.json

```diff
@@ -0,0 +1,38 @@
+{
+	name: "example-main-page"
+	image:
+	cmd: [
+		"--cas", {type: "parameter", name: "cas"},
+	]
+	requires: {
+		"cas": {
+			type: "http_service",
+			api: "cas"
+			inject: {type: "cmd", index: 3}
+		}
+		"services": {
+			type: "list"
+			of: {
+				type: "dict"
+				content: {
+					"public_path": {
+						description: "public path"
+						type: "string" // path?
```

Comment:
Możliwość dodatkowej weryfikacji. Np to, że musi być to ścieżka absolutna, albo  że musi wskazywać na istniejący endpoint.